### ISSUE TITLE
1106 Permission deny

### ISSUE THREAD

**Issue by Gatis224** [tuya/tuya-home-assistant#972]
Whenever I try to connect to my devices using the API, it generates a bug - 1106 permission deny. Making me not able to connect to my devices.

From what I've read in other topics, it has something to do with passwords and e-mail, however, I've tried different ones and none seems to be working. As far as I understand, it should be from the Tuya Smart mobile app, I've tried changing it and it still doesn't work.

The mobile app version is 5.9.0 and API is 0.6.6

**Comment by githubsebasse** [comment]
Hello, I am also interested in the answers because I have the same problem

**Comment by 78davidcross** [comment]
Yeah happening to me too.  There is no way to update the username\password from the GUI.  Editing the core.config_entries file manually doesn't work either - when the integration is reloaded the previous details are returned to the file.

**Comment by conjonway** [comment]
Yep, I'm having the same issue. Everything else appears correctly, I've double checked these trouble shooting steps too

- Your app account is not linked to your cloud project. This operation is a must-do. For more information, see [Link devices by app account](https://developer.tuya.com/en/docs/iot/Platform_Configuration_smarthome?id=Kamcgamwoevrx&_source=dc23ed1ea3e6988f278404eb2d403b1a#title-3-Link%20devices%20by%20app%20account).
- Incorrect account or password. You must enter the account and password of the mobile app that you use to scan the QR code for linking devices to your cloud project on the [Tuya IoT Development Platform](https://iot.tuya.com/?_source=a4c65f56395e05cf64cc8d4abb7396b6).
- Incorrect country. You must select the region of your account of the Tuya Smart app or Smart Life app.

**Comment by 78davidcross** [comment]
I ended up removing the integration and re-adding it again. I had to fix two entities that had been renamed in HA but otherwise it pulled everything back in from Tuya and started working straight away.

**Comment by Tobi9111** [comment]
I´ve the same issue here. Even after i have delete all (integration in HA and the whole cloud projekt in tuya IoT), than i´ve setting it up again after a full restart of homeassistant. But I can´t resolve it.

**Comment by githubsebasse** [comment]
Hello, I just followed this explanation and it works for me. I hope for you too.
Good day
https://www.lesalexiens.fr/actualites/connecter-tuya-home-assistant-facilement-2024/

**Comment by conjonway** [comment]
> Hello, I just followed this explanation and it works for me. I hope for you too. Good day https://www.lesalexiens.fr/actualites/connecter-tuya-home-assistant-facilement-2024/

Thank you, this worked great for me :)

**Comment by toto25550** [comment]
Hello,
I have the same issue 
Error 1106.
I configured as same as documentation.
Login failed: {'code': 1106, 'msg': 'permission deny', 'success': False, 't': 1737907060321, 'tid': '454ed93adbfe11efb364423a6d1d6b63'}.
I filled all information on each field.
I check and all is ok.
App mobile : smartlife.

It's ok with tuya interface but ko with tuya v2
